app/controllers/checksum_records_controller.rb

```diff
@@ -0,0 +1,53 @@
+class ChecksumRecordsController < ApplicationController
+  before_action :set_checksum_record, only: [:show, :update, :destroy]
+
+  # GET /checksum_records
+  # GET /checksum_records.json
+  def index
+    @checksum_records = ChecksumRecord.all
+  end
```

Comment:
we might also want to disallow listing all records, since there might be lots.  filed #12.